C-Plus-Plus/dp/Partition_Problem.cpp

```diff
@@ -0,0 +1,145 @@
+//Partition problem
```

Comment:
We encourage contributors to solve only 1 issue per PR, remove this file and raise a new PR from a different branch regarding this issue.

Reply:
Yes, I have raised this PR from different branch only as  Bharati2612_DFS
and for other one it is  Bharati2612_Partition.

Reply:
> Yes, I have raised this PR from different branch only as Bharati2612_DFS
> and for other one it is Bharati2612_Partition.

Remove this file from his (```Bharati2612_DFS```) branch then.